src/containers/pages/InterventionPlan/UpdatePlan/index.tsx

```diff
@@ -173,7 +178,11 @@ const UpdatePlan = (props: RouteComponentProps<RouteParams> & UpdatePlanProps) =
       <h3 className="mb-3 page-title">{pageTitle}</h3>
       <Row>
         <Col md={8}>
-          <PlanForm {...planFormProps} addPlan={fetchPlan} />
+          <PlanForm
+            actionCreator={fetchPlanRecordsCreator}
+            {...planFormProps}
+            addPlan={fetchPlan}
```

Comment:
Does it make sense to now add all of these to `planFormProps` to keep things tidy?